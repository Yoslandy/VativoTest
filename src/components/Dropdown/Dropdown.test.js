import React from "react"
import { MemoryRouter } from "react-router-dom"
import renderer from "react-test-renderer"
import { Formik, Form } from "formik"
import DropDown from "./index"

test("Renders DropDown form correctly", () => {
    const component = renderer.create(
        <MemoryRouter>
            <Formik
                initialValues={{
                    colors: "#264653",
                }}
            >
                <Form>
                    <DropDown
                        label="colors"
                        className="Test"
                        options={[
                            {
                                label: "#264653",
                                value: "#264653",
                            },
                            {
                                label: "#2a9d8f",
                                value: "#2a9d8f",
                            },
                            {
                                label: "#e9c46a",
                                value: "#e9c46a",
                            },
                        ]}
                    />
                </Form>
            </Formik>
        </MemoryRouter>
    )

    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})
