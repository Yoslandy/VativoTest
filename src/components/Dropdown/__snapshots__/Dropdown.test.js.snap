// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders DropDown form correctly 1`] = `
<form
  action="#"
  onReset={[Function]}
  onSubmit={[Function]}
>
  <div
    className="flex flex-col"
  >
    <label
      className="font-roboto text-xs text-primary-120 uppercase"
    >
      colors
    </label>
    <select
      className="dropdown  Test"
      onBlur={[Function]}
      onChange={[Function]}
      onClick={[Function]}
    >
      <option
        className="dropdown-option"
        value="#264653"
      >
        #264653
      </option>
      <option
        className="dropdown-option"
        value="#2a9d8f"
      >
        #2a9d8f
      </option>
      <option
        className="dropdown-option"
        value="#e9c46a"
      >
        #e9c46a
      </option>
    </select>
  </div>
</form>
`;
